docs/BaremetalHost_ProvisioningState.dot

```diff
@@ -5,57 +5,49 @@ digraph BaremetalHost {
 
     Unmanaged [shape=doublecircle]
     Unmanaged -> Registering [label="BMC.* != \"\""]
-    Unmanaged -> Deleting1 [label="!DeletionTimestamp.IsZero()"]
-
-    Deleting1 [shape=point]
+    Unmanaged -> PoweringOffBeforeDelete [label="!DeletionTimestamp.IsZero()"]
 
     ExternallyProvisioned [label="Externally\nProvisioned"]
 
-    Registering -> Inspecting [label="!externallyProvisioned && NeedsHardwareInspection()"]
-    Registering -> Preparing [label="!externallyProvisioned && inspectionDisabled()"]
+    Registering -> Inspecting [label="!externallyProvisioned &&\nNeedsHardwareInspection()"]
+    Registering -> Preparing [label="!externallyProvisioned &&\ninspectionDisabled()"]
     Registering -> ExternallyProvisioned [label="externallyProvisioned"]
-    Registering -> Deleting2 [label="!DeletionTimestamp.IsZero()"]
-
-    Deleting2 [shape=point]
+    Registering -> PoweringOffBeforeDelete [label="!DeletionTimestamp.IsZero()"]
```

Comment:
Same

Reply:
We already have a follow up for this here #1356